models/reporting/commissioning/mh_currencies/fct_mhsds_currency_bed_days.sql

```
@@ -0,0 +1,111 @@
+with fiscal_years as (
+    select
+        fiscal_year_start
+        , gdp_deflator
+        , case when fiscal_year_start = min(fiscal_year_start) over ()
```

SQLFluff (4.2.2)

[error] Found leading comma ','. Expected only trailing near line breaks.

(LT04)